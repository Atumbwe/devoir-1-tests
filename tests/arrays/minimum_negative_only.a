-423 -5 -3427 -323 -564 -423 -56 -324 -4234 -634 -57 -234 min