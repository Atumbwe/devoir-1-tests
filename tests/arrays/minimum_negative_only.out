 -------------
 <  -4234  >
 -------------
        \   ^__^
         \  (oo)\_______
            (__)\       )\/\
                ||----w |
                ||     ||
